CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## Upcoming release
 * Web App
   * Fixed bug that allows disabled streams to be shown & selected
+* System
+  * Manage the /boot/config.txt firmware configuration file for bugfixes
```

Comment:
I would avoid calling the software firmware, elsewhere in this repo the STM32 firmware is the only thing referred to as 'firmware'.

Reply:
Ok after another half second of thought that totally should have happened before the previous reply I'm guessing Raspberry Pi is what's calling this 'firmware configuration'. In that case, I'd just change this to 'Manage the Raspberry Pi firmware configuration file /boot/config.txt for bugfixes'.

Reply:
yeah, that definitely could be wayyy clearer; i wanna call it what they call it, but need to differentiate it. I'll modify this a bit